### PR TITLE
doc_restructuring incl. [SPOT-163]

### PR DESCRIPTION
**Pull request by NathanSegerlind**
updated documentation, added separate sections for the data schema, the jar, and the ml_ops.sh script

### CODE REVIEW

**Comment by NathanSegerlind** [comment]
I'll add some material to address SPOT-163

what we probably need somewhere is a "hey to configure your /etc/spot.conf" document

**Comment by lujacab** [comment]
+1

**Comment by raypanduro** [comment]
+1

File: spot-ml/README.md [hunk 1]
```diff
@@ -2,101 +2,40 @@
...
+spot-ml contains routines for performing *suspicious connections* analyses on netflow, DNS or proxy data gathered from a network. These
+analyses consume collections of network events and produce lists of the events that are considered to be the least probable (most suspicious).
 
-spot-ml is designed to be run as a component of Spot. It relies on the ingest component of Spot to collect and load
-netflow and DNS records, and spot-ml will try to load data to the operational analytics component of Spot.  It is suggested that when experimenting with spot-ml, you do so as a part of the unified Spot system: Please see [the Spot wiki]
+The suspicious connects analysis assigns to each record a score in the range 0 to 1, with smaller scores being viewed as more suspicious or anomalous than larger scores.
 
-The remaining instructions in this README file treat spot-ml in a stand-alone fashion that might be helpful for customizing and troubleshooting the
-component.
+These routines are contained in a jar file   and there is a shell script ml_ops.sh for a simplified the invocation of the analyses.
```
Comment (brandon-edwards) [inline spot-ml/README.md]: There seems to be a typo here on new line 10, "...for a simplified the invocation..."

**Comment by brandon-edwards** [comment]
+1 (one type pointed out)

**Comment by brandon-edwards** [comment]
'typo' that is

**Pull request merged**